python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -156,3 +154,41 @@ def compile(model: nn.Module,
                 return pl_model
         else:
             return pl_model
+
+    def quantize(self, model, calib_dataloader, val_dataloader=None, metric: str = None,
+                 backend='inc', conf=None, framework='pytorch_fx', approach='ptsq',
+                 strategy='bayesian', accuracy_criterion=None, timeout=0, max_trials=1):
+        if backend == 'inc':
+            from bigdl.nano.quantization import QuantizationINC
+            quantizer = QuantizationINC(framework=framework, conf=conf, approach=approach,
+                                        strategy=strategy, accuracy_criterion=accuracy_criterion,
+                                        timeout=timeout, max_trials=max_trials)
+            q_litmodel = copy.deepcopy(model)
+            quantizer.model = q_litmodel.model
+            q_approach = quantizer.cfg.quantization.approach
+
+            def eval_func(model_to_eval):
+                if val_dataloader:
+                    q_litmodel.model = model_to_eval
```

Comment:
`model_to_eval` is a Pytorch Module instead of PL Module. I want to reuse PL API 'evaluate(val_dataloader)' here to evaluate the performance of current quantized model.

Reply:
A PL Module don't necessarily has a member variable `model`; and a PL Module is also a PyTorch `nn.module`, no?

Reply:
Right, PL module is a `nn.module`. Let me try if I can call `trainer.evaluate(quantized_model.model, ...)`.

Reply:
Failed to use PL module directly, since our implementation of `forward` in LightingModuleFromTorch is incompatible with pytorch FX.
```python
def forward(*args, **kwargs):
     return self.model(*args, **kwargs)
```
During graph tracing, `args` and `kwargs` are a Proxy object that can't be parsed by `*`.

Reply:
I think the point is that you cannot assume the `model` (or `q_model`) here is `LightingModuleFromTorch`; it can be a conventional `LightingModule` (or even a conventional `nn.Module`), and then the above code will not work.

Reply:
Yes, I agree. I'm trying to figure out a quantization pipeline to cover both cases (use-defined and nano-provided pl module).